parallel-install/pkg/deployment/deployment.go

```diff
@@ -53,43 +63,95 @@ type Installer interface {
 //See overrides.New for details about the overrides contract.
 //
 //resourcesPath is a local filesystem path where components' charts are located.
-func NewDeployment(prerequisites [][]string, componentsYaml string, overridesYamls []string, resourcesPath string, cfg config.Config, processUpdates chan<- ProcessUpdate) (*Deployment, error) {
+func NewDeployment(prerequisites [][]string, componentsYaml string, overridesYamls []string, resourcesPath string, cfg config.Config, processUpdates chan<- ProcessUpdate, kubeClient kubernetes.Interface) (*Deployment, error) {
```

Comment:
Would recommend to move this to the config.Config struct. The method signature is already very long. Please see #166 for the refactored signature.

Maybe worth to think about moving parts which are coupled to the deployment-package / simple parameters to the config object (e.g. also the componentListPath, Overries). This would have the benefit to add parameter-validation to the config-object to identify missing parameters.

Reply:
we agreed to leave it like that for now and deal with it in following PR